### DISCUSSION TITLE
Phalcon documentation inifnite redirect bug

### DISCUSSION THREAD

**Discussion by shovv** [phalcon/cphalcon#16481] [General]
Hi, there is bug with inifnite redirects

![image](https://github.com/phalcon/cphalcon/assets/58041110/a1b4918a-3e9e-4658-9d0c-34b958faa836)

For example if you visit documentation page from SERP you get what you see on screenshot above. If you enter documentation directly from Phalcon page it works.

**Comment by niden** [comment]
Thank you. Also reported here: https://github.com/phalcon/documentation/issues/5

Working on it.
